doc/tools/create_changelog.py

```diff
@@ -0,0 +1,161 @@
+#!/usr/bin/python
+from __future__ import absolute_import
+from __future__ import unicode_literals
+from __future__ import print_function
+import sys
+import io
+from subprocess import check_output, CalledProcessError
```

Comment:
Certainly I'm not a Pythonist, but ```from``` seems to be akin to c++ ```using``` which more often than not feels evil to me.  Would it kill you to write ```subprocess.check_output```?